src/components/HelpTopicLink/HelpTopicLink.tsx

```diff
@@ -9,16 +9,9 @@ type HelpTopicLinkProps = {
 
 export const HelpTopicLink: React.FC<HelpTopicLinkProps> = ({ topicId, isInline, children }) => {
   const {
-    helpTopics: { setActiveTopic, enableTopics, disableTopics },
+    helpTopics: { setActiveTopic },
   } = useChrome();
 
-  React.useEffect(() => {
-    enableTopics(topicId);
```

Comment:
I am curious why wasn't this working? One render of each link it should enable the help topic and add it to the list. If I remember correctly, this used to work before.

Reply:
To be honest, I couldn't recall if it use to work or not. I just saw it not working now. The name `enableTopics` to me did not indicate that it would be additive.
If it was working before we can investigate if chrome has changed.